python/run-tests

```diff
@@ -96,6 +98,20 @@ function run_ml_tests() {
 
 function run_streaming_tests() {
     echo "Run streaming tests ..."
+
+    KAFKA_ASSEMBLY_DIR="$FWDIR"/external/kafka-assembly
+    JAR_PATH="${KAFKA_ASSEMBLY_DIR}/target/scala-${SPARK_SCALA_VERSION}"
+    for f in "${JAR_PATH}"/spark-streaming-kafka-assembly-*.jar; do
+      if [[ ! -e "$f" ]]; then
+        echo "Failed to find Spark Streaming Kafka assembly jar in $KAFKA_ASSEMBLY_DIR" 1>&2
+        echo "You need to build Spark with sbt streaming-kafka-assembly/assembly or mvn package" \
+              "before running this program" 1>&2
+        exit 1
+      fi
+      KAFKA_ASSEMBLY_JAR="$f"
+    done
+
+    export EXTRA_PYSPARK_SUBMIT_ARGS="-v --jars ${KAFKA_ASSEMBLY_JAR}"
```

Comment:
Do you need the `-v` ? With that i think the test out will become unnecessarily verbose